projects/ngneat/helipopper/src/lib/tippy.directive.ts

```diff
@@ -202,7 +202,16 @@ export class TippyDirective implements OnChanges, AfterViewInit, OnDestroy, OnIn
         this.globalConfig.onCreate?.(instance);
       },
       onShow: instance => {
-        this.zone.run(() => this.instance.setContent(this.resolveContent()));
```

Comment:
1. We should not create the instance if the value is null or undefined.
2. We should listen to changes and perform a re-check.

Reply:
I don't know if option two is so common. Maybe I'd just start with not creating the instance if [tippy] is nil

Reply:
1) The idea was to keep as much existing behaviour and use the documented behaviour of the onShow callback. Do you think it is more appropriate to guard the creation of the instance itself? I will make some modifications. `createInstance` is used three times. I think I am going to guard the method call... Do you have any thoughts on this?

2) I am not sure either, but I will take a look into it.

Reply:
Addressed 1) in the last commit.